### ISSUE TITLE
wvaughn9

### ISSUE THREAD

**Issue by bvaughn8** [RoboJackets/jr#43]
# Week 7: Git Challenge

Welcome to week 7 of software training! This week, you'll be completing a series of `git` and GitHub challenges. **To get started, type your GT username into the Title box above, then click the "Submit new issue" at the bottom of this page!** From there, work through the rest of this checklist.

If you need any help, we recommend these online resources:
- https://help.github.com/articles/github-glossary/
- https://try.github.io/levels/1/challenges/1
- https://learngitbranching.js.org/
- https://chris.beams.io/posts/git-commit/

You can, of course, also flag down an instructor if you run into anything.

## Your objectives

Replace `gburdell3` with your GT username anywhere it appears in this list.

- [X] 1. Star this repository.
- [X] 2. Create a fork of this repository.
- [X] 3. Clone your fork to your computer.
- [X] 4. Set your forked repository as a remote named `origin`.
- [X] 5. Set our repository as a remote named `rj`.
- [X] 6. Create a branch named your GT username (ex. `wvaughn9`).
- [X] 7. Use the terminal to create a folder named `2017` in the root of the repo.
- [X] 8. Use the terminal to create an empty text file named `wvaughn9.txt` in the `2017` folder.
- [X] 9. Stage and commit the file (make sure you leave good commit messages!)
- [X] 10. Create a new branch from `wvaughn9` named `wvaughn9-add-text`.
- [X] 11. Add some ASCII art to your file (no longer than 30 lines, no more than 80 characters per line) on that branch and merge it into the `gburdell3` branch. Use spaces, not tabs, for indentation in your art, as tabs may distort your image in jr.
- [X] 12. Make a merge conflict happen between `wvaughn9` and a branch named `wvaughn9-merge-conflict`.
- [X] 13. Resolve the merge conflict.
- [X] 14. Create two branches named `b1` and `b2`.
- [X] 15. Commit a file called `stuff1.txt` in `b1`.
- [X] 16. Commit a file called `stuff2.txt` in `b2`.
- [X] 17. Take the changes from `b1` and apply them to `b2`.
- [X] 18. In `b2`, delete both files and commit the changes.
- [X] 19. Merge `b2` into `b1`.
- [X] 20. Merge `b1` into `wvaughn9` (this should not cause a merge conflict).
- [X] 21. Send your committed changes to the `wvaughn9` branch on your fork
- [ ] 22. Create a pull request from your fork’s `wvaughn9` branch to our master branch and ask one of the instructors in your session to approve it (if you try to merge it you won’t be able to because permissions). **In your pull request description, add a line that says "Fixes #xx" where #xx is the number of the issue you created.** This will auto-close your issue once your pull request is merged.
- [ ] 23. After your PR has been approved, delete all local branches except for `master` and `gburdell3`.
